lib/modules/dosomething/dosomething_metatag/dosomething_metatag.module

```diff
@@ -121,8 +121,11 @@ function dosomething_metatag_get_permalink_vars() {
   $rbid = arg(1);
   $fid = ($_GET['fid']);
   $rb_entity = reportback_load($rbid);
-  if (!$fid) {
-    $fid = $rb_entity->fids[0];
+  if (!$fid && !empty($rb_entity->fids)) {
+    $fid = array_pop($rb_entity->fids);
+  }
+  else {
+    $fid = array_pop($rb_entity->getFids());
```

Comment:
So I find that if i hardcode the rbid to an integer in `dosomething_metatag_get_permalink_vars` and call it within `devel/php`, I see the fids array when i output it.  So i think there is something real weird happening taking place calling the `reportback_load` in a `hook_preprocess_page`.

I also tried outputting `vars` in there, to see if the loaded reportback was already in there, but it's not. i think if the template was using some default Entity template type stuff vs a theme function, we'd get those variables for free in the `$vars`.   The theme function seems to be using `$rb` as just an array, not a loaded `reportback` entity..